sdks/python/apache_beam/io/fileio.py

```diff
@@ -521,6 +523,10 @@ def _rename_file(rename_op):
         return(None, e)
       return (final_name, None)
 
+    # ThreadPool crashes in old versions of Python (< 2.7.5) if created from a
```

Comment:
This seems like a different thing. Isn't it? Better do it in a separate CL unless it blocks the inprocess runner.